lib/commands/run-api-scenario.ts

```diff
@@ -122,6 +135,25 @@ export const builder: yargs.CommandBuilder = {
 
 export async function handler(argv: yargs.Arguments): Promise<void> {
   await cliSuppressExceptions(async () => {
+    const swaggerFilePaths: string[] = (argv.specs || []).map((it: string) => path.resolve(it));
+    if (argv.readme !== undefined) {
+      const readmeMd: string = path.resolve(argv.readme);
+      const autorestConfig = await getAutorestConfig(argv, readmeMd);
```

Comment:
if `tag` is specified, we should load swagger files under it, otherwise, load the default tag.

Reply:
Yes, it in the argv, and passed to autorest